src/renderer/pages/settings/index.js

```
@@ -152,9 +152,42 @@ function initializeUI() {
  * Initialize Cloud Sync UI
  */
 function initializeCloudSyncUI() {
-  console.log('initializeCloudSyncUI 호출');
+  console.log('initializeCloudSyncUI 호출 - 구독:', authState.subscription ? authState.subscription.plan : 'none');
 
   try {
+    // premium 구독 사용자는 항상 Cloud Sync 기능 활성화
+    if (authState.subscription && authState.subscription.plan) {
+      const plan = authState.subscription.plan.toLowerCase();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)